docs/usage/shoot_settings/shoot_autoscaling.md

```diff
@@ -21,21 +22,21 @@ Consequently, please refer to the [official documentation](https://github.com/ku
 
 The `Shoot` API allows to configure a few flags of the `cluster-autoscaler`:
 
-There are [general options for `cluster-autoscaler`](../api-reference/core.md#core.gardener.cloud/v1beta1.ClusterAutoscaler), and these values will be used for all worker groups except for those overwriting them. Additionally, there are some [`cluster-autoscaler` flags to be set per worker pool](../api-reference/core.md#core.gardener.cloud/v1beta1.ClusterAutoscalerOptions). They override any general value such as those specified in the general flags above.
+There are [general options for `cluster-autoscaler`](../../api-reference/core.md#core.gardener.cloud/v1beta1.ClusterAutoscaler), and these values will be used for all worker groups except for those overwriting them. Additionally, there are some [`cluster-autoscaler` flags to be set per worker pool](../../api-reference/core.md#core.gardener.cloud/v1beta1.ClusterAutoscalerOptions). They override any general value such as those specified in the general flags above.
 > Only some `cluster-autoscaler` flags can be configured per worker pool, and is limited by NodeGroupAutoscalingOptions of the upstream community Kubernetes repository. This list can be found [here](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/config/autoscaling_options.go#L37-L55).
 
 ## Vertical Pod Auto-Scaling
 
 This form of auto-scaling is not enabled by default and must be explicitly enabled in the `Shoot` by setting `.spec.kubernetes.verticalPodAutoscaler.enabled=true`.
-The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually.
+The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Also, if `ShootVPAEnabledByDefault` admissionPlugins is enabled, then `.spec.kubernetes.verticalPodAutoscaler.enabled` will be set to true.
```

Comment:
```suggestion
The reason is that it was only introduced lately, and some end-users might have already deployed their own VPA into their clusters, i.e., enabling it by default would interfere with such custom deployments and lead to issues, eventually. Additionally, if the `ShootVPAEnabledByDefault` admission plugin is enabled in the gardener-apiserver, then the `.spec.kubernetes.verticalPodAutoscaler.enabled` field will be defaulted to true.
```

Add also a link to the admission plugin docs in `apiserver-admission-plugins.md`.